README.md

```diff
@@ -1,5 +1,5 @@
 <p align="center">
-  <img width = "50%" src='docs/img/habitat_logo_with_text_horizontal_blue.png' />
+  <img width = "50%" src='res/img/habitat_logo_with_text_horizontal_blue.png' />
```

Comment:
Do we need rename or move `docs/img` to `res/img`?

Reply:
I had moved the imgs folder from `docs/` to a `res/` folder under the main repository, hence this renaming is needed. Does this answer your question?

Reply:
ow, finally found that in changed file list.